src/main/python/pgshovel/streams/utilities.py

```diff
@@ -24,3 +32,33 @@ def __str__(self):
             active=','.join(map(str, active[:self.max])),
             truncated='' if len(active) <= self.max else ',+{0}...'.format(len(active) - self.max),
         )
+
+
+def is_start_batch_operation(message):
+    """
+    Is the ``message`` the start operation of a batch of mutations.
+    """
+    value = get_oneof_value(message.batch_operation, 'operation')
+    return type(value) in TRANSACTION_START_EVENT_TYPES
+
+
+def prime_until(stream, check, max_messages):
```

Comment:
I'd switch the order here so that `max_messages` is the first argument (since it is the most likely to be `partial`, and reads a little cleaner.)